Automattic/simplenote-ios
# Renames OptionsViewController > SettingsViewController

### Details:
- We're gonna need to introduce **NoteOptionsViewController**
- And **OptionsViewController** actually displays **Simplenote's Settings**

Because of the above... sending a tiny housekeeping PR!

@diegoreymendez Sir!! can I bug you with this one?
THankThanks in advance!!

### Test
1. Log into your account
2. Press over the top left button
3. Press over the `Settings` row

- [ ] Verify the Simplenote Settings UI shows up onscreeen

### Release
These changes do not require release notes.
